pkg/services/provisioning/dashboards/file_reader.go

```diff
@@ -412,42 +413,139 @@ type dashboardIdentity struct {
 }
 
 func (d *dashboardIdentity) Exists() bool {
-	return len(d.title) > 0 && d.folderID > 0
+	return len(d.title) > 0
```

Comment:
Please illuminate me, why is `&& d.folderID > 0` no longer necessary?

Reply:
It's a bug. folderID `0` is an ID of the General folder. We need to remove this clause to track duplicates in the General folder too.

Reply:
Thanks for clarifying that!